packages/jest-haste-map/src/index.js

```diff
@@ -230,6 +231,18 @@ class HasteMap {
     });
   }
 
+  matchGlobs(globs: Array<Glob>, rootDir: Path): Promise<Array<Path>> {
+    return this.build().then(hasteMap => {
+      const files = [];
```

Comment:
we should really start using Set for this stuff instead of arrays. Mind starting the trend?